cmd/flash/src/lib.rs

```diff
@@ -305,6 +306,10 @@ fn flashcmd(
                             c = Some("STM32G030C6Tx".to_string());
                             break;
                         }
+                        if nr.is_match(s) {
```

Comment:
Unrelated to the PR description.

Reply:
Sorry, those snuck in because I committed from the master branch, classic github mistake.